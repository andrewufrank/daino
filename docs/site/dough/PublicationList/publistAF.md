---
title: publistAF.md
author: auf
date: 2010-07-29
keywords: homepage
pageTemplate: page3.yaml
abstract: |
    produce a publist for af

bibliographyGroup: authorAF
style: "/home/frank/Workspace8/ssg/docs/site/dough/resources/chicago-fullnote-bibliography-bb.csl"
bibliography: "/home/frank/Workspace8/ssg/docs/site/dough/resources/BibTexLatex.bib"


---

An example publication list for AF  -- but not the relative "resources/BibTexLatex.bib"

 
