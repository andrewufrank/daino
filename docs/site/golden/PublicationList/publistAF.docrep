Docrep {meta1 = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", dyLink = "PublicationList/publistAF.md", dyLang = DLenglish, dyTitle = "publistAF.md", dyAbstract = "produce a publist for af", dyAuthor = "auf", dyDate = Just "2010-07-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Just "resources/chicago-fullnote-bibliography-bb.csl", dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", link = "PublicationList/publistAF.md", title = "publistAF.md", abstract = "produce a publist for af", author = "auf", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, pan1 = Pandoc (Meta {unMeta = fromList [("abstract",MetaBlocks [Para [Str "produce",Space,Str "a",Space,Str "publist",Space,Str "for",Space,Str "af"]]),("author",MetaInlines [Str "auf"]),("bibliography",MetaInlines [Str "resources/BibTexLatex.bib"]),("bibliographyGroup",MetaInlines [Str "authorAF"]),("date",MetaInlines [Str "2010-07-29"]),("keywords",MetaInlines [Str "homepage"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("style",MetaInlines [Str "resources/chicago-fullnote-bibliography-bb.csl"]),("title",MetaInlines [Str "publistAF.md"])]}) [Para [Str "An",Space,Str "example",Space,Str "publication",Space,Str "list",Space,Str "for",Space,Str "AF",Space,Str "--",Space,Str "but",Space,Str "not",Space,Str "the",Space,Str "relative",Space,Str "\"resources/BibTexLatex.bib\""]]}