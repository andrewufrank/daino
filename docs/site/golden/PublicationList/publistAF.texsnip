TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", dyLink = "PublicationList/publistAF.md", dyLang = DLenglish, dyTitle = "publistAF.md", dyAbstract = "produce a publist for af", dyAuthor = "auf", dyDate = Just "2010-07-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Just "resources/chicago-fullnote-bibliography-bb.csl", dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", link = "PublicationList/publistAF.md", title = "publistAF.md", abstract = "produce a publist for af", author = "auf", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "An example publication list for AF -\\/- but not the relative\n\"resources/BibTexLatex.bib\""}