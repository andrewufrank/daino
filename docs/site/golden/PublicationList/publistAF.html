<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="auf" />
              <meta name="dcterms.date" content="2010-07-29" />
              <meta name="keywords" content="homepage" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">publistAF.md</h1>
        
                <p class="author">auf</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=PublicationList/publistAF.html><em>publistAF.md </em></a>
				    <br>produce a publist for af		     
				    <span class=tiny>auf "2010-07-29" 
            PublicationList/publistAF.html Nothing
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- blog text content section -->
    
    abstractproduce a publist for af 
    content-from-panrep-blocks<p>An example publication list for AF -- but not the relative "resources/BibTexLatex.bib"</p>
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
